components/Sections/ImageTextColumns.tsx

```diff
@@ -0,0 +1,59 @@
+import { getImageInfo } from "@/lib/getAssetInfo";
+import prisma, { Prisma } from "@/lib/prisma";
+import Image from "next/image";
+import React from "react";
+
+// ToDo: It crashes when the leftImage or rightImage is null
+
+export type ImageTextColumnsProps = Prisma.ImageTextColumnsGetPayload<{}>;
+
+export async function ImageTextColumns({
+  title,
+  content,
+  leftImage,
+  rightImage,
+}: ImageTextColumnsProps) {
+  const infoLeft = await getImageInfo(String(leftImage))
+    .then((res) => res)
+    .catch((err) => null);
+  const infoRight = await getImageInfo(String(rightImage))
+    .then((res) => res)
+    .catch((err) => null);
```

Comment:
Don't call these functions in the UI component. Make slots for left and right image and fetch them in the builder. You can make new components for fetching and showing the image.